### PR TITLE
Structural changes

### PR DESCRIPTION
**Pull request by inamoto4**
Please check carefully.

### CODE REVIEW

File: data/helper/data_dictionary.csv [hunk 1]
```diff
@@ -35,9 +35,9 @@ Text,publication_date,date of first publication in original language,,2017-07-04
 Text,publishing_house,name of publishing house,provide pinyin for Chinese publishing houses,2017-07-04,LH,2017-07-04,LH
 Text,first_chin_edition,year of first edition in China,,2017-07-04,LH,2017-07-04,LH
 Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,,,,
-Text,neibu,yes/no whether this was originally a neibu publication,,2017-07-04,LH,2017-07-04,LH
+Text,neibu,"yes/no whether this was originally a neibu publication, add „0“ for every publication starting from 1980",,2017-07-04,LH,2017-07-04,LH
```
Comment (duncdrum) [inline data/helper/data_dictionary.csv]: avoid the use of U+201E/F  ``„‟`` in english, it messes up the auto-parsing of tables, use two U+0027 (singel-quote) ``''`` inside text cells, or U+0022 ``""`` if you absolutely have to use multiple nested quotation marks.

File: data/Text.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-text_id,title,subtitle,title_lang,person_id,genre_id,publication_place,publication_date,publishing_house,first_chin_edition,physical_description,neibu,source_id,page_number,note,created,created_by,last_modified,last_modified_by
+text_id,title,subtitle,title_lang,person_id,genre_id,publication_place,publication_date,publishing_house,first_chin_edition,physical_description,neibu,source_id,pages,note,created,created_by,last_modified,last_modified_by
```
Comment (duncdrum) [inline data/Text.csv]: first_chin_edition is still in file see #31 ?

File: data/Person.csv [hunk 3]
```diff
@@ -1,337 +1,337 @@
...
-P0178,Darwin,Charles,en,NULL,NULL,NULL,NULL,1809,1882,,,PL0085,,NULL,https://de.wikipedia.org/wiki/Charles_Darwin,,2017-10-08,MG,2017-10-09,MG
-P0178,达尔文,查尔斯,zh,NULL,NULL,NULL,NULL,1809,1882,,,PL0085,,NULL,https://de.wikipedia.org/wiki/Charles_Darwin,,2017-10-08,MG,2017-10-09,MG
-P0179,Crankshaw ,Edward ,en,NULL,NULL,NULL,NULL,1909,1984,,,PL0087,,NULL,https://en.wikipedia.org/wiki/Edward_Crankshaw,,2017-10-08,MG,2017-10-09,MG
-P0179,MIRA,,zh,NULL,NULL,NULL,NULL,1909,1984,,,PL0087,,NULL,https://en.wikipedia.org/wiki/Edward_Crankshaw,,2017-10-08,MG,2017-10-09,MG
-P0180,Song,Yongyi,en,,,,,1949,,,,PL0003,,,https://fr.wikipedia.org/wiki/Song_Yongyi,,2017-10-09,MG,2017-10-09,MG
-P0180,宋,永毅,zh,,,,,1949,,,,PL0003,,,https://fr.wikipedia.org/wiki/Song_Yongyi,,2017-10-09,MG,2017-10-09,MG
+person_id,family_name,first_name,name_lang,sex,rustication,time_of_rustication_start,time_of_rustication_end,place_of_rust,birthyear,deathyear,alt_name,alt_name_lang,place_of_birth,social_position,neibu_access,source_id,page_number,note,created,created_by,last_modified,last_modified_by
```
Comment (duncdrum) [inline data/Person.csv]: please include references to issues when addressing them in a PR

File: data/Text.csv [hunk 4]
```diff
@@ -1,186 +1,186 @@
...
-T00108,“A Glance at the Underground Reading Movement during the Cultural Revolution”,,en,P0180,G19,PL0084,2007,Journal of Contemporary China,,"Vol. 16, no. 51, 325-333",,NULL,NULL,,2017-10-09,MG,2017-10-09,MG
-T00109,Memorial for Not Forgetting,remembering My friend Zhu Shixian,en,P0031,G19,,2010,Remembrance,,no. 53 July 15,,NULL,NULL,,2017-10-09,MG,2017-10-09,MG
-T00109,为了不忘却的纪念,忆难友朱时贤,zh,P0031,G19,,2010,Remembrance,,no. 53 July 15,,NULL,NULL,,2017-10-09,MG,2017-10-09,MG
+text_id,title,subtitle,title_lang,person_id,genre_id,publication_place,publication_date,publishing_house,first_chin_edition,physical_description,neibu,source_id,pages,commentary,note,created,created_by,last_modified,last_modified_by
```
Comment (duncdrum) [inline data/Text.csv]: *yay* for first_chin_edition, see #31

File: data/helper/data_dictionary.csv [hunk 5]
```diff
@@ -33,12 +35,13 @@ Text,genre_id,Refer to genre IS,,2017-07-04,LH,2017-07-04,LH
...
 Text,publishing_house,name of publishing house,provide pinyin for Chinese publishing houses,2017-07-04,LH,2017-07-04,LH
-Text,first_chin_edition,year of first edition in China,,2017-07-04,LH,2017-07-04,LH
-Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,,,,
-Text,neibu,yes/no whether this was originally a neibu publication,,2017-07-04,LH,2017-07-04,LH
+Text,first_chin_edition,"year of first edition in China. This is a nice-to-have category, please do not invest too much time in finding out! We will keep it for the moment, but the contained information may be deleted or copied elsewhere.",,2017-07-04,LH,2017-07-04,LH
+Text,physical_description,please add information in two cases: 1) if you have any information about the physical condition of the object 2) for internet source add: 'Internet Media Type',,2017-10-12,OS,2017-10-12,OS
+Text,neibu,"yes/no whether this was originally a neibu publication, add „NULL“ for every publication starting from 1980",,2017-07-04,LH,2017-07-04,LH
```
Comment (duncdrum) [inline data/helper/data_dictionary.csv]: please avoid the german U+201E and U+201F characters ``„‟``, they technically don't exist in english. Most importantly, they often interfere with the parsing of the csv file. Use two U+0027 ``'`` inside a cell, or if nested quotes are necessary U+0022 ``"``

**Comment by inamoto4** [comment]
I am still not sure about the physical description. The text entry is a BIBLIOGRAPHICAL entry as in library catalogues and does actually not relate to a specific object! "torn in half" can therefore not be a valid value, right?

**Comment by duncdrum** [comment]
It certainly can, when library catalogs record the size and medium of a book, it can include "written on cigarette paper" ,"damaged", or what not.

**Comment by inamoto4** [comment]
Yes, but the information "damaged" refer to the specific object a library holds. We are not a library, we are sth more like Worldcat, I would say.

**Comment by duncdrum** [comment]
You don't have to hold an object to describe it, world-cat will display physical description data just the same as a local catalogue. You could add ``NULL`` to make it clear that you are recording a fictional book, without real world counterpart. This should not be confused with the physical description of webpages, which are virtual, but very much real. Similarly you could reserve ``   ``` (empty) or use ``unknown`` to express that you do not know anything about the physical properties.

File: data/Text.csv (hunk 2 above)
Comment (inamoto4) [reply data/Text.csv]: Yeah, I thought we are about to keep it as an additional information?

File: data/helper/data_dictionary.csv (hunk 5 above)
Comment (inamoto4) [reply data/helper/data_dictionary.csv]: #104

**Pull request merged**